apps/challenges/serializers.py

```diff
@@ -63,11 +67,47 @@ class ChallengePhaseSplitSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = ChallengePhaseSplit
-        fields = '__all__'
         fields = ('id', 'dataset_split', 'challenge_phase', 'challenge_phase_name', 'dataset_split_name', 'visibility')
 
     def get_dataset_split_name(self, obj):
         return obj.dataset_split.name
 
     def get_challenge_phase_name(self, obj):
         return obj.challenge_phase.name
+
+
+class DatasetSplitSerializer(serializers.ModelSerializer):
```

Comment:
I am guessing that you moved the `DatasetSplitSerializer` here so that all the serializers are in alphabetical order, right?